rosdep/python.yaml

```diff
@@ -689,6 +689,18 @@ python-evdev:
     vivid: [python-evdev]
     wily: [python-evdev]
     xenial: [python-evdev]
+python-expiringdict:
+  debian:
+    jessie: [python-expiringdict]
+    sid: [python-expiringdict]
+    stretch: [python-expiringdict]
+  fedora:
+    pip:
+      packages: [expiringdict]
+  ubuntu:
+    xenial: [python-expiringdict]
+    yakkety: [python-expiringdict]
```

Comment:
Also here calling out the version is not necessary since it's the same for all versions.